lib/Common/Memory/HeapBucket.h

```diff
@@ -267,6 +268,10 @@ template <bool pageheap, typename Fn>
 void
 HeapBucketT<TBlockType>::SweepBucket(RecyclerSweep& recyclerSweep, Fn sweepFn)
 {
+    if (recyclerSweep.IsBackground()) 
```

Comment:
I think you need to zero this out in SetupBackgroundSweep to catch all the page during background sweep.
Otherwise, you won't 0 it out until the background thread gets to it?